django/urls/resolvers.py

```diff
@@ -190,7 +190,7 @@ def _compile(self, regex):
         except re.error as e:
             raise ImproperlyConfigured(
                 '"%s" is not a valid regular expression: %s' % (regex, e)
-            )
+            ) from e
```

Comment:
If we're already nitpicking, do we need to have the original error message in the new exception if we're also linking the original exception?

Reply:
Agreed and fixed.

Reply:
I think we should keep the previous behavior. An original exception was always in a traceback. It's easier for users to have it in the most recent message and don't force them to search for a real cause in the traceback.